deploy/helm/values.yaml

```diff
@@ -649,6 +649,12 @@ cluster:
   # Useful for example if multiple clusters have to have the same name. Or when the name contains problematic characters.
   uid: ""
 
+  filter:
+    exclude_namespaces: []
+    exclude_namespaces_regex: []
+    include_namespaces: []
+    include_namespaces_regex: []
+
```

Comment:
`values.yaml` serves also as documentation of the options, so please describe the usage in the comment (especially the condition that only one of these can be set)